src/mixins/filing-template-mixin.ts

```diff
@@ -382,9 +393,11 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     this.setBusinessSnapshot(businessSnapshot)
 
     // Restore current entity type
+    // TODO: should we omit this as it was set from business snapshot ???
```

Comment:
If you can answer this question, either fix it or create a ticket to fix it.
Also on line 400 below.
Thanks!

Reply:
Any takers?